internal/consensus/replay_test.go

```diff
@@ -339,7 +339,7 @@ func setupSimulator(ctx context.Context, t *testing.T) *simulatorTestSuite {
 		nPeers,
 		"replay_test",
 		newMockTickerFunc(true),
-		newPersistentKVStoreWithPath)
+		newPersistentKVStore)
```

Comment:
It might be a little cleaner to use an argument struct for all of these and just have the default case of this function create the ephemeral kv store.

Reply:
yeah, I don't disagree, but I think this isn't the right place to clean up this detail